app/src/main/java/io/gnosis/safe/ui/transactions/TransactionListFragment.kt

```diff
@@ -80,8 +80,10 @@ class TransactionListFragment : SafeOverviewBaseFragment<FragmentTransactionList
 
         binding.transactions.addOnScrollListener(object : RecyclerView.OnScrollListener() {
             override fun onScrolled(recyclerView: RecyclerView, dx: Int, dy: Int) {
-                val firstVisibleItem = (binding.transactions.layoutManager as LinearLayoutManager).findFirstVisibleItemPosition()
-                reload = firstVisibleItem <= 1
+                if (lifecycle.currentState.isAtLeast(Lifecycle.State.STARTED)) {
```

Comment:
I guess checking the lifecycle state is the right thing to do, but why is this not already on master?

Reply:
this was relevant for release, so it was fixed here, to produce new rc